doc/SecurityScan.md

```diff
@@ -1,5 +1,7 @@
 # MobSF Security scan integration
 
+This project includes automated static application security testing (SAST) for the generated binaries for both iOS and Android. This is helpful to identify opportunities to reduce your app's susceptibility to attacks from malicious third parties. We use [MobSF](https://github.com/MobSF/Mobile-Security-Framework-MobSF) to scan the produced artifacts and generate a report with the analysis results.
```

Comment:
Nitpick: In markdown, I like to put every sentence on a new line because it makes it easier to read (especially when reviewing changes in PRs) and it also does not affect the rendered version.

Reply:
Made the adjustments